sdk/core/azure-core/src/test/java/com/azure/core/util/CoreUtilsTests.java

```diff
@@ -75,6 +76,20 @@ public void bomAwareToString(byte[] bytes, String contentType, String expected)
         assertEquals(expected, CoreUtils.bomAwareToString(bytes, contentType));
     }
 
+    @ParameterizedTest
+    @MethodSource("validApplicationId")
```

Comment:
If you name "validateApplicationId" parameter the same as the test method name, you don't need to specify it. The framework can infer what the arguments are. And you can use `@methodSource`